week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -15,8 +15,27 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 
 function sortAges(arr) {
-
-}
+   let numberArr =[];// new array to store numbera
+  for(let i = 0; i < arr.length;i++){ // loops throw the length of an array
+        if(typeof arr[i]!== typeof'string'){// checks if array elements are number
+           numberArr.push(arr[i]);// push the numbers into new array
+        }
+      }
+      let swapped; // variable to hold a swap
+      for(let j= numberArr.length; j > 0;j--){// loops though the array desending order

```

Comment:
We have a convention (not a rule! Nothing goes wrong if you don't follow it, but it helps other programmers understand your code quickly when reading it) that we start loop counters from `i`, and go on in the alphabet in inner loops.

Because we tend to use `i` for the outer-most `for` loop, when I first read this code I assumed that it was inside another loop (which uses `i`). This definitely isn't a programming rule, or anything wrong, but because that's a convention we have, it may be more clear to other readers to use `i` and `j` here, rather than `j` and `k`